src/libs/utillib/conversions.cpp

```diff
@@ -12,6 +12,9 @@
  *-------------------------------------------------------------------------------------------*/
 #include "biglib.h"
 #include "conversions.h"
+#include <iomanip> // setprecision
+#include <sstream> // stringstream
```

Comment:
Just a coding style thing. I usually put the system headers in a file called ./src/libs/utillib/basetypes.h (these two may already be included there).

If I do put system headers in a .cpp file, I put them first.

The reason is because the linter will tell me to do that when (and if) we run the linter.